components/proxy/plugins/configcat/configcat.go

```diff
@@ -64,19 +68,43 @@ func (ConfigCat) CaddyModule() caddy.ModuleInfo {
 	}
 }
 
+func ServeFromFile(w http.ResponseWriter, r *http.Request, fileName string) {
+	fp := path.Join(configCatConfigDir, fileName)
+	d, err := os.Stat(fp)
+	if err != nil {
+		w.Write(DefaultConfig)
```

Comment:
Are configcat clients ok with receiving empty configs? Will they throw if they cannot find the key they're looking for? Is there some common way ConfigCat responds on errors?